website/src/components/LeaderboardTable/LeaderboardTable.tsx

```diff
@@ -63,18 +63,21 @@ export const LeaderboardTable = ({
       },
       columnHelper.accessor("display_name", {
         header: t("user"),
-        cell: ({ getValue, row }) => (
-          <div className="flex flex-row items-center gap-2">
-            <UserAvatar displayName={getValue()} avatarUrl={row.original.image} />
-            {isAdminOrMod ? (
-              <Link as={NextLink} href={`/admin/manage_user/${row.original.user_id}`}>
-                {getValue()}
-              </Link>
-            ) : (
-              getValue()
-            )}
-          </div>
-        ),
+        cell: ({ getValue, row }) => {
+          const display_name = getValue();
+          return (
+            <div className="flex flex-row items-center gap-2">
+              <UserAvatar key={display_name} displayName={display_name} avatarUrl={row.original.image} />
```

Comment:
We already have an effect in the `UserAvatar` component, so why do we need key here?

Reply:
I wanted to be _really sure_ that it gets changed, I will remove this.